src/sort.js

```diff
@@ -45,11 +45,30 @@ module.exports = function (list) {
     },
   }
 
+  function isFloatString(value) {
+    if (value === undefined || value === null || !value || typeof value !== 'string' || value.split(".").length !== 2) {
```

Comment:
```javascript
    value.split(".").length !== 2
```
--> just check value(string type) has only one `.` because of `ip` sorting test case.